### PR TITLE
Add description for napari hub

### PR DESCRIPTION
**Pull request by DragaDoncila**
Add .napari/DESCRIPTION.md for use on napari hub.

Note the gif is too large for github to render...

### CODE REVIEW

File: .napari/DESCRIPTION.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+This GUI plugin allows you to quickly find the affine matrix mapping
+one image to another using manual correspondence points annotation.
+
+More simply, this plugin allows you to select corresponding points
+on an image, and a second image you wish to transform. It computes 
+the requisite transformation matrix using Affine Transform, Euclidean Transform, 
+or Similarity Transform, and performs this transformation on the
+moving image, aligning it to the reference image.
+
+![Affinder aligning two images of a parasitic worm](https://i.imgur.com/FIhnqa0.gif)
```
Comment (jni) [inline .napari/DESCRIPTION.md:12]: Please remove this gif.

😜😜😜😜😜

File: .napari/DESCRIPTION.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+![Adding corresponding points to newly focused layer](https://i.imgur.com/JVZCvmp.png)
+
+affinder will immediately transform the moving image to align the points you've
+selected when you add your third corresponding point to your moving image.
+
+![The moving image is transformed once three points are added](https://i.imgur.com/NTne9fj.png)
+
+From there, you can continue iteratively adding sets of three points until you 
+are happy with the alignment. Click Finish to exit affinder.
```
Comment (jni) [inline .napari/DESCRIPTION.md:61]: No, once you have three it alternates after every point

File: .napari/DESCRIPTION.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+- [**Affine Transform**](https://en.wikipedia.org/wiki/Affine_transformation): 
+the least rigid transformation, it preserves
+lines and parallelism, but not necessarily distance and angles. Translation,
+scaling, similarity, reflection, rotation and shearing are all valid
+affine transformations.
+
+- [**Similarity Transform**](https://en.wikipedia.org/wiki/Similarity_(geometry)): 
+this is a "shape preserving" transformation, producing objects which are 
+geometrically similar. Translation, rotation, reflection and scaling are
```
Comment (jni) [inline .napari/DESCRIPTION.md:79]: scaling -> uniform scaling, or single scaling (as opposed to different scaling along different axes)

File: .napari/DESCRIPTION.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+- [**Euclidean Transform**](https://en.wikipedia.org/wiki/Rigid_transformation):
+Also known as a rigid transformation, this transform preserves the Euclidean
+distance between each pair of points on the image. This includes rotation,
+translation and reflection but not scaling or shearing.
+
+# Getting Help
+
+If you find a bug with affinder, or would like support with using it, please raise an
+issue on the GitHub repository `https://github.com/jni/affinder`.
```
Comment (jni) [inline .napari/DESCRIPTION.md:90]: This should be a link rather than plaintext.

**Review by jni — CHANGES_REQUESTED** [review]
Just requested some minor changes @DragaDoncila! Oh and if you could contact the W3C and GitHub to fix the video embedding problem that'd be just great.

great

File: .napari/DESCRIPTION.md (hunk 1 above)
Comment (DragaDoncila) [reply .napari/DESCRIPTION.md]: ApprehensiveTornIndigobunting-small

**Comment by DragaDoncila** [comment]
@jni made those changes, I think this is ready now.

**Comment by jni** [comment]
@DragaDoncila I was totally joking about the gif but thanks for delivering! 😂 🙏

**Review by jni — APPROVED** [review]

**Pull request merged**